benchmarks/src/main/scala/zio/IODeepLeftBindBenchmark.scala

```diff
@@ -34,9 +34,9 @@ class IODeepLeftBindBenchmark {
 
   def zioDeepLeftBindBenchmark(runtime: Runtime[Any]): Int = {
     var i  = 0
-    var io = IO.succeedLazy(i)
+    var io = IO.succeed(i)
```

Comment:
This should become `effectTotal` in line with other effect types that delay execution in their benchmarks (e.g. monix `Task.eval`)

Reply:
Fixed